src/ast/rewriter/seq_rewriter.cpp

```diff
@@ -1963,6 +1954,184 @@ bool seq_rewriter::is_sequence(expr* e, expr_ref_vector& seq) {
     return true;
 }
 
+br_status seq_rewriter::mk_regexp_contains_emptystr(expr* b, expr_ref& result) {
```

Comment:
is_nullable takes care of this

Reply:
Yes, I kept it around in case we want it later, but now that you have created the PR I will remove it.